base/applications/mspaint/winproc.cpp

```diff
@@ -879,8 +878,6 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
                 HBITMAP hbmNew = DoLoadImageFile(m_hWnd, szFileName, FALSE);
                 if (hbmNew)
                     InsertSelectionFromHBITMAP(hbmNew, m_hWnd);
-                else
-                    ShowError(IDS_LOADERRORTEXT, szFileName);
```

Comment:
Is this error redundant because it's already triggered by DoLoadImageFile perhaps?

Reply:
`DoLoadImageFile` displays error message on error.